src/ray/common/ray_config_def.h

```diff
@@ -565,7 +565,7 @@ RAY_CONFIG(uint64_t, subscriber_timeout_ms, 300 * 1000)
 RAY_CONFIG(uint64_t, gcs_actor_table_min_duration_ms, /*  5 min */ 60 * 1000 * 5)
 
 /// Whether to enable GCS-based actor scheduling.
-RAY_CONFIG(bool, gcs_actor_scheduling_enabled, false);
```

Comment:
we need a roll out/benchmark plan before turn it on by default.

Reply:
Yes. It is only used to test all cases with gcs scheduler enabled. I'll turn if off after code review.